LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -248,19 +300,22 @@ function addHeader(column) {
             });
             break;
 	    default:
-	    	if(column.substr(0, 6) == 'depID_') { // backwards compatibility for LEAF workflow requirement based approval dates
-	    		depID = column.substr(6);
+	    	if(column.substr(0, 6) === 'depID_') { // backwards compatibility for LEAF workflow requirement based approval dates
+	    		let depID = column.substr(6);
```

Comment:
depID should be passed through at line 326, so I dont see the reason for this let? This might cause issues changing how its previously working. Also the next if statement for stepID is set the same way before as well. On line 329.
```suggestion
	    		depID = column.substr(6);
```

Reply:
Going to change this just is case per our discussion.  stepID I hadn't touched, because there are some variables defined in the file without first being declared to suggest that they are used globally, and I wasn't confident that this value wasn't also being used elsewhere.